File: segment-sniper-react/src/models/Segment/SummarySegment.ts
export interface SummarySegment {
  id: string;
  name: string;
  activityType: string;
  distance: number;
  averageGrade: number;
  maximumGrade: number;
  elevationHigh: number;
  elevationLow: number;
  startLatlng: [number, number];
  endLatlng: [number, number];
  elevationProfile: string | null;
  climbCategory: number;
  city: string;
  state: string;
  country: string;
  isPrivate: boolean;
  isHazardous: boolean;
  isStarred: boolean;
}

File: SegmentSniper.Models/Strava/Segment/SummarySegment.cs
namespace SegmentSniper.Models.Strava.Segment
{
    public class SummarySegment
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public string ActivityType { get; set; }
        public double Distance { get; set; }
        public double AverageGrade { get; set; }
        public double MaximumGrade { get; set; }
        public double ElevationHigh { get; set; }
        public double ElevationLow { get; set; }
        public List<double> StartLatlng { get; set; }
        public List<double> EndLatlng { get; set; }
        public object ElevationProfile { get; set; }
        public int ClimbCategory { get; set; }
        public string City { get; set; }
        public string State { get; set; }
        public string Country { get; set; }
        public bool Private { get; set; }
        public bool Hazardous { get; set; }
        public bool Starred { get; set; }
    }
}
